components/saa-server/src/main/java/com/wso2telco/util/DBConnection.java

```diff
@@ -61,9 +61,31 @@ public static DBConnection getInstance() throws ClassNotFoundException {
      * @throws DBUtilException DbUtilException
      */
     public void addClient(String clientDeviceId, String platform, String pushToken, String msisdn) throws SQLException, DBUtilException {
+
+        logger.info("Register Client : ClientDeviceID"+clientDeviceId+",Platform : "+platform+",PushToken : "+pushToken+",MSISDN : "+msisdn+"");
+        Connection connection = null;
+        PreparedStatement statement = null;
         String timeStamp = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(Calendar.getInstance().getTime());
-        String query = "INSERT INTO clients (client_device_id,platform,push_token,date_time,msisdn) VALUES ('" + clientDeviceId + "','" + platform + "','" + pushToken + "','" + timeStamp + "','" + msisdn + "');";
-        executeUpdate(query);
+        String query = "INSERT INTO clients (client_device_id,platform,push_token,date_time,msisdn) VALUES (?,?,?,?,?);";
+
+        try {
+            connection = getConnection();
+            statement = connection.prepareStatement(query);
+            statement.setString(1,clientDeviceId);
+            statement.setString(2,platform);
+            statement.setString(3,pushToken);
+            statement.setString(4,timeStamp);
+            statement.setString(5,msisdn);
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            logger.error("SQLException occurred !!!" + e);
+            throw new SQLException(e.getMessage(), e);
```

Comment:
better to add a meaningful msg here rather than e.getMessage() as anyway it's getting printed in the stack trace
eg: "Error occurred while inserting client details for Client Id : " + clientDeviceId